charts/rancher-eks-controller/0.1.000/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-eks-controller
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/system-charts/charts/eks-controller
+sources:
+  - "https://github.com/rancher/system-charts/charts/eks-controller"
+version: 0.1.000
```

Comment:
Doesn't need to be triple zeroes, you can just do 0.1.0

Reply:
Denise requested this